driver/gpio/imxgpio/imx6ull.hpp

```diff
@@ -0,0 +1,577 @@
+
+    { IMX_MAKE_PIN_1(1, 17), 0, 0x624, 3 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RX_DATA - ALT0 - UART1_RX - IOMUXC_UART1_RX_DATA_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 17), 2, 0x5B8, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RX_DATA - ALT2 - I2C3_SDA - IOMUXC_I2C3_SDA_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 17), 3, 0x4C8, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RX_DATA - ALT3 - CSI_DATA03 - IOMUXC_CSI_DATA03_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 17), 4, 0x594, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RX_DATA - ALT4 - GPT1_CLK - IOMUXC_GPT1_CLK_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 17), 8, 0x618, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RX_DATA - ALT8 - SPDIF_IN - IOMUXC_SPDIF_IN_SELECT_INPUT
+
+    { IMX_MAKE_PIN_1(1, 18), 0, 0x620, 2 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_CTS_B - ALT0 - UART1_CTS_B - IOMUXC_UART1_RTS_B_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 18), 2, 0x66C, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_CTS_B - ALT2 - USDHC1_WP - IOMUXC_USDHC1_WP_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 18), 3, 0x4D8, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_CTS_B - ALT3 - CSI_DATA04 - IOMUXC_CSI_DATA04_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 18), 8, 0x69C, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_CTS_B - ALT8 - USDHC2_WP - IOMUXC_USDHC2_WP_SELECT_INPUT
+
+    { IMX_MAKE_PIN_1(1, 19), 0, 0x620, 3 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RTS_B - ALT0 - UART1_RTS_B - IOMUXC_UART1_RTS_B_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 19), 2, 0x668, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RTS_B - ALT2 - USDHC1_CD_B - IOMUXC_USDHC1_CD_B_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 19), 3, 0x4CC, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RTS_B - ALT3 - CSI_DATA05 - IOMUXC_CSI_DATA05_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 19), 8, 0x674, 2 }, // IOMUXC_SW_MUX_CTL_PAD_UART1_RTS_B - ALT8 - USDHC2_CD_B - IOMUXC_USDHC2_CD_B_SELECT_INPUT
+
+    { IMX_MAKE_PIN_1(1, 20), 0, 0x62C, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA - ALT0 - UART2_TX - IOMUXC_UART2_RX_DATA_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 20), 2, 0x5BC, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA - ALT2 - I2C4_SCL - IOMUXC_I2C4_SCL_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 20), 3, 0x4DC, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA - ALT3 - CSI_DATA06 - IOMUXC_CSI_DATA06_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 20), 4, 0x58C, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA - ALT4 - GPT1_CAPTURE1 - IOMUXC_GPT1_CAPTURE1_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 20), 8, 0x560, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA - ALT8 - ECSPI3_SS0 - IOMUXC_ECSPI3_SS0_B_SELECT_INPUT
+
+    { IMX_MAKE_PIN_1(1, 21), 0, 0x62C, 1 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_RX_DATA - ALT0 - UART2_RX - IOMUXC_UART2_RX_DATA_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 21), 2, 0x5C0, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_RX_DATA - ALT2 - I2C4_SDA - IOMUXC_I2C4_SDA_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 21), 3, 0x4E0, 0 }, // IOMUXC_SW_MUX_CTL_PAD_UART2_RX_DATA - ALT3 - CSI_DATA07 - IOMUXC_CSI_DATA07_SELECT_INPUT
+    { IMX_MAKE_PIN_1(1, 21), 4, 0x590, 0 }, // IOMUXC_S- ALT4 - GPT1_CAPTURE2 - IOMUXC_GPT1_CAPTURE2_SELECT_INPUT

```

Comment:
The first item in the description comment is missing the MUX_CTL_PAD value, did the script for parsing the NXP documentation fail here, or was this truncated by accident?